 #This error is caused when you write a index which is not present in the list
stuff=["Nithin","Abin","Local train"]
# print(stuff[3])    This would give an Indexerror





